OWML.Common/IModConfig.cs

```diff
@@ -10,7 +10,5 @@ public interface IModConfig
 
         T GetSettingsValue<T>(string key);
         void SetSettingsValue(string key, object value);
-
-        IModConfig Copy();
```

Comment:
This was only being used to create the snapshot, which is now only needed in ModMergedConfig, not in defaultConfig or userConfig